routers/users.py

```
@@ -0,0 +1,165 @@
+    user_id = current_user["id"]
+
+    posts = data.load_data("posts.json")
+    user_post_ids = [p["id"] for p in posts if p.get("user_id") == user_id]
+
+    comments = data.load_data("comments.json")
+    comments = [c for c in comments
+                if c.get("user_id") != user_id and c.get("post_id") not in user_post_ids]
+    data.save_data(comments, "comments.json")
+
+    likes = data.load_data("likes.json")
+    likes = [l for l in likes
```

Ruff (0.14.11)

Ambiguous variable name: `l`

(E741)